app/assets/stylesheets/src/shared/_components.scss

```diff
@@ -175,7 +175,7 @@
   &.tiny {
     --sc_font_size: 12px;
     --sc_min_height: 28px;
-    --sc_chevron_size: 0.4em;
+    --sc_chevron_size: 5.3px;
```

Comment:
these weird values come because we transform the chevron by 45deg

Reply:
![image](https://github.com/user-attachments/assets/f4e92dad-3fa9-4eed-ac21-926ef1bf6599)
as you see width is 7.5, and height is half of it: 3.75